core/object/script_language.cpp

```diff
@@ -460,6 +459,51 @@ void ScriptLanguage::get_core_type_words(List<String> *p_core_type_words) const
 
 void ScriptLanguage::frame() {
 }
+TypedArray<int> ScriptLanguage::CodeCompletionOption::get_option_characteristics(String base) {
+	// return characacteristics of the match found by order of importance
+	// matches will be ranked by a lexicographical order on the vector returned by this function
+	// the lower values indicate better matches and that they should go before in the order of appearance
```

Comment:
```suggestion
	// Return characteristics of the match found by order of importance.
	// Matches will be ranked by a lexicographical order on the vector returned by this function.
	// The lower values indicate better matches and that they should go before in the order of appearance.
```
Likewise for other comments.